app/InterAppCommunication/NSString+IACExtensions.m

```diff
@@ -0,0 +1,71 @@
+//
+//  NSString+IACExtensions.m
+//  IACSample
+//
+//  Created by Antonio Cabezuelo Vivo on 10/02/13.
+//  Copyright (c) 2013 Antonio Cabezuelo Vivo. All rights reserved.
+//
+
+#import "NSString+IACExtensions.h"
+
+#if !__has_feature(objc_arc)
+#error InterAppComutication must be built with ARC.
+// You can turn on ARC for only InterAppComutication files by adding -fobjc-arc to the build phase for each of its files.
+#endif
+
+
+@implementation NSString (IACExtensions)
+
++ (NSString*)stringWithUUID {
```

Comment:
I'm not convinced this pulls its weight, especially since the alternative isn't particularly onerous (`[NSUUID UUID].UUIDString`)

Reply:
I'm not the "InterAppCommunication" developer. It's an existing Cocoapods library by Antonio Cabezuelo Vivo